File: src/uk/co/armedpineapple/corsixth/wizard/WelcomeWizard.java
/*
 *   Copyright (C) 2012 Alan Woolley
 *   
 *   See LICENSE.TXT for full license
 */
package uk.co.armedpineapple.corsixth.wizard;

import uk.co.armedpineapple.corsixth.Configuration;
import uk.co.armedpineapple.corsixth.ConfigurationException;
import android.content.Context;
import android.util.AttributeSet;

public class WelcomeWizard extends WizardView {

	public WelcomeWizard(Context context, AttributeSet attrs, int defStyle) {
		super(context, attrs, defStyle);
	}

	public WelcomeWizard(Context context, AttributeSet attrs) {
		super(context, attrs);

	}

	public WelcomeWizard(Context context) {
		super(context);

	}

	@Override
	void saveConfiguration(Configuration config) throws ConfigurationException {

	}

	@Override
	void loadConfiguration(Configuration config) {

	}

}

File: src/Java/uk/co/armedpineapple/cth/wizard/WelcomeWizard.kt
/*
 *   Copyright (C) 2012 Alan Woolley
 *   
 *   See LICENSE.TXT for full license
 */
package uk.co.armedpineapple.cth.wizard

import uk.co.armedpineapple.cth.Configuration
import uk.co.armedpineapple.cth.Configuration.ConfigurationException
import android.content.Context
import android.util.AttributeSet

class WelcomeWizard : WizardView {

    constructor(context: Context, attrs: AttributeSet, defStyle: Int) : super(context, attrs, defStyle) {}

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs)

    constructor(context: Context) : super(context)

    @Throws(ConfigurationException::class)
    override fun saveConfiguration(config: Configuration) {

    }

    override fun loadConfiguration(config: Configuration) {

    }

}
